src/core/lib/security/credentials/tls/grpc_tls_credentials_options.h

```diff
@@ -65,18 +71,36 @@ struct grpc_tls_credential_reload_config
       void (*destruct)(void* config_user_data));
   ~grpc_tls_credential_reload_config();
 
+  void* context() const { return context_; }
+  void set_context(void* context) { context_ = context; }
+
   int Schedule(grpc_tls_credential_reload_arg* arg) const {
+    if (schedule_ == nullptr) {
+      gpr_log(GPR_ERROR, "schedule API is nullptr");
+      return 1;
```

Comment:
Same as above. We also need to populate `arg` field appropriately.

Reply:
Also, how about doing this check together with `arg != nullptr`? and remove the `context_` check below (set `config` field always)? The comment applies to other methods and configs.